src/utils/data-types.ts

```diff
@@ -64,7 +66,7 @@ extend(NUMBER, "number");
 extend(STRING, "string");
 extend(DATE, "datetime");
 extend(BOOLEAN, "boolean");
-extend(ARRAY, "array");
-extend(OBJECT, "object");
+// extend(ARRAY, "array");
```

Comment:
there is no array type right now - instead there is an isArray property, but I didn't cover arrays at all in this PR - maybe in some time we will make an addition